### ISSUE TITLE
Research creating email accounts with an address like info@accesslocator.com

### ISSUE THREAD

**Issue by joshi1983** [hhaccessibility/hhaccessibility.github.io#82]
If it is possible, make one associated with Blandine's gmail account.

**Comment by joshi1983** [comment]
@Blandine-AA , I came across [some documentation](https://www.digitalocean.com/community/tutorials/how-to-set-up-gmail-with-your-domain-on-digitalocean) on how to link the domain to gmail email services and there is a 30 day trial period.  The trial would be followed by a $5 a month service charge for a Google Apps account.  Do you want to get this through JMCC?

A free alternative would be that I set up email service on our digital hosting account.  This would be free of money but could take hours of my time that would otherwise be directed at code development on the application.  This alternative would also require time spent upgrading, troubleshooting on my end whereas gmail would be maintained by Google and probably be much more reliable.

**Comment by Blandine-AA** [comment]
Hi Josh,

no need for a paid service. I think once the info account is created, I will be able to access it from gmail. Otherwise, we will use webmail that comes with the domain name

**Comment by joshi1983** [comment]
I'll check with GoDaddy's support service to look into this more and confirm if you're right about this being a part of the domain names we purchased from GoDaddy.  If they aren't, the previous comment outlines our main options.

**Comment by joshi1983** [comment]
I tried to get help from GoDaddy on Saturday but their chat service was unavailable and I didn't want to call.  I'll try again soon.

**Comment by joshi1983** [comment]
I confirmed with GoDaddy and here is my conversation with GoDaddy's agent, Kirk.  We didn't purchase any emailing service with the domain names.  Like mentioned 2 weeks ago, we could pay for email service from a few sources and that would be the best solution.  We can pay GoDaddy or Google for email service associated with our domain names.  Blandine, you mentioned knowing someone who promptly got email working with gmail.  If you think he/she can help find an alternative, let them contact me.

> 
Josh Greig:
What do you recommend for a way to get a working info@accesslocator.com where accesslocator.com is one of the domains purchased through GoDaddy?
Kirk:
Are you Guys just going to create 1 email only for the domain?
Josh Greig:
It may be a few. info@accesslocator.com is the only one I'm aware of for now.
Kirk:
Okay then well you Guys can use the Office 365 Email Essentials that we have, that has a 5 GB of email capacity as a basic email and works on both IMAP/POP settings and it is also working on a separate server so there is a 100% assurance of 0 downtime.
Josh Greig:
sounds good
How much extra would that cost?
Kirk:
It will be only for $3.99 per month minimum of 1 year Registration.
Josh Greig:
ok
Would you sign in to view email through the GoDaddy site?
sort of like logging in with hotmail or gmail
Kirk:
No it will have a separate Webmail link and yes the email can be associated to email apps like Outlook, Apple mail, Gmail and Thunder bird.
Josh Greig:
ok
Lastly, I just want to confirm that the current purchases of domain names such as accesslocator.com, accesslocator.info... didn't include this email service. Can you confirm?
The transaction would have been made around November 21, 2016.
Kirk:
No email service that has been purchase along the with the domain registrations.
Josh Greig:
ok so it'll be the extra $3.99 per month for minimum of one year, right?
Kirk:
Yes that's correct.

**Comment by Blandine-AA** [comment]
Hi Josh,
I talked to my friend and the email is created through Bluehost, who is the web hosting company.
So, can you check with Digital Ocean? They should also provide a limited number of email addresses as it seems to be a standard in the industry.
Thanks,

**Comment by joshi1983** [comment]
What we paid for with GoDaddy was domain names which don't come with email service.  What we paid for with Digital Ocean is a virtual private server which is not email service unless I set up a mail server on it which isn't recommended.  We're talking about $4 or $5 per month for email service or about $50 per year so our time spent discussing and my time spent researching a free alternative doesn't balance well with the dollar figures.  I've already discussed with other web developers, coworkers, GoDaddy, and researched online.  Looking more seems like a waste of time to me.

If info@accesslocator.com is worth $50 or so per year, let's ask JMCC to cover it.

**Comment by areebakamal** [comment]
Meeting:

*Ask JMCC if we can point to their email server for accesslocator.com email accounts such as admin@accesslocator.com.

**Comment by joshi1983** [comment]
I assigned this to Blandine since it is waiting on her to talk with JMCC about shared email as Areeba commented.

**Comment by Blandine-AA** [comment]
Got JMCC answer and it is NO for now.

Let's work with the gmail account for now and we will see later to get a separate webmail server or figure out another solution with JMCC or OACRS.

**Comment by joshi1983** [comment]
This research has been conducted and we have our answers so I'm closing this issue.  Another issue will be made if and when @accesslocator.com email gets the money and priority required for implementation.
